concordium-consensus/src/Concordium/KonsensusV1/TreeState/StartUp.hs

```diff
@@ -247,8 +247,8 @@ loadSkovData _genesisBlockHeight _runtimeParameters didRollback = do
             && not consensusIsShutdown
             then case mLatestFinEntry of
                 Nothing ->
-                    throwM . TreeStateInvariantViolation $
-                        "Missing finalization entry for last finalized block"
+                    -- In this case, by the above check, the last finalized block is the genesis block.
+                    return (blockEpoch lastFinBlock, Absent)
```

Comment:
The referenced above check is:
```haskell
        case mLatestFinEntry of
            Nothing
                | blockRound lastFinBlock == 0 ->
                    return
                        CertifiedBlock
                            { cbQuorumCertificate = genesisQuorumCertificate (getHash genesisBlock),
                              cbQuorumBlock = genesisBlock
                            }
                | otherwise ->
                    throwM . TreeStateInvariantViolation $
"Missing finalization entry for last finalized block"
```
i.e. if the round wasn't 0 (indicating genesis) we'd have already thrown an exception.